src/libs/actions/Report.js

```diff
@@ -432,6 +432,35 @@ function unsubscribeFromReportChannel(reportID) {
     Pusher.unsubscribe(pusherChannelName);
 }
 
+/**
+ * Get the report ID for a chat report for a specific
+ * set of participants and redirect to it.
+ *
+ * @param {String[]} participants
+ */
+function fetchOrCreateChatReport(participants) {
+    if (participants.length < 2) {
+        throw new Error('fetchOrCreateChatReport() must have at least two participants');
+    }
+
+    API.CreateChatReport({
+        emailList: participants.join(','),
+    })
+
```

Comment:
Oops, I just noticed this after merging... this is an extra empty line here that should be removed.

Reply:
Good catch. Tagged you in PR to remove it here: https://github.com/Expensify/Expensify.cash/pull/1483